nncf/common/utils/backend.py

```diff
@@ -174,3 +178,15 @@ def is_openvino_available() -> bool:
     :return: True if openvino package is installed, False otherwise.
     """
     return _OPENVINO_AVAILABLE
+
+
+def is_openvino_at_least(version_str: str) -> bool:
+    """
+    Check if OpenVINO version is at least the specified one.
+    :param version_str: The version string to compare with the installed OpenVINO version. For example "2025.1".
+    :return: True if the installed OpenVINO version is at least the specified one, False otherwise.
```

Comment:
```suggestion
    Check if OpenVINO version is at least the specified one.
    
    :param version_str: The version string to compare with the installed OpenVINO version. For example "2025.1".
    :return: True if the installed OpenVINO version is at least the specified one, False otherwise.
```